### DISCUSSION TITLE
I'm trying to add spacing to image in blog article but it's not working properly.

### DISCUSSION THREAD

**Discussion by dmxt** [gatsbyjs/gatsby#37253] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

## Default example for one blog article
![image](https://user-images.githubusercontent.com/3248782/207462266-d442d4c7-163d-4c05-aa19-9049de3ad5c8.png)
_It's shrunk down by default._

## What I want to do with this image
**I want to:** give a margin (or padding) around the image and perhaps some box shadow, but Gatsby is making it impossible for me. 

### Look, guys
I don't want to do a hack like manually adding padding and box-shadow around the image, which CSS is perfectly capable of. I didn't expect this from Gatsby.

---

Before I made this report, I wanted to let you guys know that I'm good at inspecting and CSS, so I tried many things but to no avail. 

I found an HTML above this image: a blank white rectangle 20x6 big. Without this element, the image I wanted to post wouldn't appear. I do not know the reasoning behind it.

---

### Reproduction Link

My project: https://github.com/dmxt/dmxt-blog
The initial link I pulled from: https://github.com/gatsbyjs/gatsby-starter-blog

### Steps to Reproduce

1. Clone and start https://github.com/dmxt/dmxt-blog
2. Go to `~/content/blog/disqus-on-gatsby`
3. In this folder, you will find one `index.md` file, one picture file inside the `images` folder
4. Try adding spacing around your image, like `margin` or `padding`. I wish you good luck in debugging this.

(I couldn't even do it inside Chrome's inspect element, which I'm very familiar with, there is some severe confusion)

### Expected Result

Normal CSS response. Adding `margin: 50px` to an image will make a space, but...

### Actual Result

Abnormal CSS response I've never seen before. **The image is overlapping something else.** I need someone to explain this to me.

### Environment

```shell
System:                                                    
    OS: Windows 10 10.0.19045 (I'M TEMPORARILY ON WINDOWS, NOT MACOS)                              
    CPU: (4) x64 Intel(R) Core(TM) i5-4670K CPU @ 3.40GHz    
  Binaries:                                                  
    Node: 18.12.1 - C:\Program Files\nodejs\node.EXE         
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD 
    npm: 8.19.2 - C:\Program Files\nodejs\npm.CMD            
  Browsers:                                                  
    Edge: Spartan (44.19041.1266.0), Chromium (108.0.1462.46)
  npmPackages:                                               
    gatsby: ^5.0.1 => 5.0.1
    gatsby-plugin-feed: ^5.0.0 => 5.0.0
    gatsby-plugin-image: ^2.23.0 => 2.24.0
    gatsby-plugin-manifest: ^4.23.0 => 4.24.0
    gatsby-plugin-sharp: ^4.23.0 => 4.24.0
    gatsby-remark-copy-linked-files: ^5.23.0 => 5.24.0
    gatsby-remark-images: ^6.23.0 => 6.24.0
    gatsby-remark-prismjs: ^6.23.0 => 6.24.0
    gatsby-remark-responsive-iframe: ^5.23.0 => 5.24.0
    gatsby-remark-smartypants: ^5.23.0 => 5.24.0
    gatsby-source-filesystem: ^4.23.0 => 4.24.0
    gatsby-transformer-remark: ^5.23.0 => 5.24.0
    gatsby-transformer-sharp: ^4.23.0 => 4.24.0
```
_I use macOS, too; please don't hate me_

**Comment by LekoArts** [comment]
I'm not really following what exact problem you have 🤔 If you want to have styles applied to the image, apply CSS to the class `.gatsby-resp-image-wrapper`
